src/main/java/org/cascadebot/cascadebot/commands/music/VolumeCommand.java

```diff
@@ -0,0 +1,55 @@
+package org.cascadebot.cascadebot.commands.music;
+
+import net.dv8tion.jda.core.entities.Member;
+import org.cascadebot.cascadebot.commandmeta.CommandContext;
+import org.cascadebot.cascadebot.commandmeta.ICommandMain;
+import org.cascadebot.cascadebot.commandmeta.Module;
+import org.cascadebot.cascadebot.music.CascadePlayer;
+import org.cascadebot.cascadebot.permissions.CascadePermission;
+
+public class VolumeCommand implements ICommandMain {
+
+    @Override
+    public void onCommand(Member sender, CommandContext context) {
+        CascadePlayer player = context.getData().getMusicPlayer();
+        if (context.getArgs().length == 0) {
+            context.getTypedMessaging().replyInfo("Current volume is %s%%", player.getPlayer().getVolume());
+            return;
+        }
+        Integer Arguments = Integer.valueOf(context.getMessage(0));
```

Comment:
1. why is this name Argument`s`
2. camelCase

Reply:
oh also you should parse this better.
first off make the variable an int second off parse it with Integer.parseInt() and surround that in a try catch

Reply:
`why is this name Arguments` I don't know what else to name it to. Suggestion?

Reply:
Why is getMessage being used? We have a get arg as int method

Reply:
also you can name it `volume` :P